### PR TITLE
GMLCity Importer

### PR DESCRIPTION
**Pull request by nicolasbandel**
Issue #161
Issue #346

### CODE REVIEW

File: backend/src/BIE.DataPipeline/DbHelper.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ internal sealed class DbHelper
         private readonly StringBuilder mStringBuilder;
 
         private int mCount;
-        private const int MaxCount = 900;
+        private const int MaxCount = 1;//900;
```
Comment (Corgam) [inline backend/src/BIE.DataPipeline/DbHelper.cs:15]: Should it be changed?

File: backend/src/BIE.DataPipeline/yaml/building_models.yaml
Comment (Corgam) [inline backend/src/BIE.DataPipeline/yaml/building_models.yaml:1]: Please follow the new yaml folder structure, so either put these files into `common`, `developement` or `production` subfolders. Did you merge into your branch the latest `sprint-release`?

**Review by Corgam — CHANGES_REQUESTED** [review]
Looks good, except two things

File: backend/src/BIE.DataPipeline/DbHelper.cs (hunk 1 above)
Comment (nicolasbandel) [reply backend/src/BIE.DataPipeline/DbHelper.cs]: No you are right I have reverted it

File: backend/src/BIE.DataPipeline/yaml/building_models.yaml
Comment (nicolasbandel) [reply backend/src/BIE.DataPipeline/yaml/building_models.yaml:1]: I moved the filles in the folders.

**Review by Corgam — APPROVED** [review]
Looks good, thanks for fixing the comments!

**Pull request merged**